e2e/test/config/TestConfig.cs

```diff
@@ -1,14 +1,13 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System.Diagnostics.Tracing;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
 
-[assembly: Parallelize(Workers = 1, Scope = ExecutionScope.ClassLevel)]
+[assembly: Parallelize(Workers = 0, Scope = ExecutionScope.ClassLevel)]
```

Comment:
Why 0 workers?

Reply:
That means that the test engine will use as many workers as there are available cores on the computer running it. I'll add an explanation in the comments